mint/bessy/bessy_interface.py

```diff
@@ -186,10 +186,10 @@ def get_quick_add_devices(self):
                               "QUAD:LI26:501:BCTRL"]),
             ("LI26 601-901", ["QUAD:LI26:601:BCTRL", "QUAD:LI26:701:BCTRL", "QUAD:LI26:801:BCTRL",
                               "QUAD:LI26:901:BCTRL"]),
-            ("LTU M. Quads", ["QUAD:LTU1:620:BCTRL", "QUAD:LTU1:640:BCTRL", "QUAD:LTU1:660:BCTRL",
-                              "QUAD:LTU1:680:BCTRL"]),
+            ("LTU M. Quads", ["QUAD:LTUH:620:BCTRL", "QUAD:LTUH:640:BCTRL", "QUAD:LTUH:660:BCTRL",
+                              "QUAD:LTUH:680:BCTRL"]),
             ("Dispersion Quads", ["QUAD:LI21:221:BCTRL", "QUAD:LI21:251:BCTRL", "QUAD:LI24:740:BCTRL",
-                                  "QUAD:LI24:860:BCTRL", "QUAD:LTU1:440:BCTRL", "QUAD:LTU1:460:BCTRL"]),
```

Comment:
Maybe should not touch the `bessy_interface` despite it looking like a copy of the LCLS Machine Interface.

Reply:
Addressed via https://github.com/ocelot-collab/optimizer/pull/30/commits/eb8e8d91e232a26a7ddf629956c0212f5894b577